types/skatejs/index.d.ts

```diff
@@ -0,0 +1,28 @@
+// Type definitions for skatejs 5.0
+// Project: https://github.com/skatejs/skatejs
+// Definitions by: Martin Hochel <https://github.com/Hotell>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.5
```

Comment:
Having TypeScript Version 2.5 here implies that these typings are incompatible with versions less than 2.5. But I don't see anything here that would make that true. Unless I'm missing something, you're better off removing this line so that more users can use these typings.

Reply:
valid, downgrading to 2.3 as default generics are used